npm-scripts.js

```diff
@@ -17,6 +17,21 @@ const MAKE = process.env.MAKE || (isFreeBSD ? 'gmake' : 'make');
 // eslint-disable-next-line no-console
 console.log(`npm-scripts.js [INFO] running task "${task}"`);
 
+if (isWindows)
+{
+	if (!fs.existsSync('worker/out/msys/bin/make.exe'))
+	{
+		installMsysMake();
+	}
+
+	const msysPath = `${process.cwd()}\\worker\\out\\msys\\bin`;
+
+	if (!process.env['PATH'].includes(msysPath))
+	{
+		process.env['PATH'] = `${msysPath};${process.env['PATH']}`;
+	}
+}
```

Comment:
This block is being executed no matter which task is given as argument to `npm-scripts.js`. I don't think this is acceptable. Can we out this into a function and just call it by tasks that really require this?